### PR TITLE
feat(devops):added build-iossimulator

### PR DESCRIPTION
**Pull request by cnlangzi**
### Changes
- added build-iossimulator to build zcncore.xcframework for iOS simulators

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
-

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -117,4 +117,4 @@ require (
 )
 
 //replace github.com/ethereum/go-ethereum => github.com/certifaction/go-ethereum v1.10.3-wasm
-//replace github.com/herumi/bls-go-binary => github.com/0chain/bls-go-library-zboxmobile v1.26.1
+replace github.com/herumi/bls-go-binary => github.com/cnlangzi/bls-go-binary v1.26.1
```
Comment (peterlimg) [inline go.mod]: Any changes in `github.com/cnlangzi/bls-go-binary'? if so, let's fork the bls-go-binary to 0chain org, and commit the changes.

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (cnlangzi) [reply go.mod]: - fixed simulator issue. see detail on https://github.com/herumi/bls-go-binary/pull/9
- uploaded pre-complied binaries for iOS and iOS simulator on `binary` branch, and create a tag for it.

Comment (cnlangzi) [reply go.mod]: it seems be forked on https://github.com/0chain/bls-go-library-zboxmobile by @moldis.  I am not sure if it is fine to merge them. let me submit a PR on bls-go-library-zboxmobile

Comment (cnlangzi) [reply go.mod]: fixed

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**